build/follow-me/render.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/**
+ * Server-side rendering of the `activitypub/follow-me` block.
+ *
+ * @package ActivityPub
+ */
+
+use Activitypub\Blocks;
+
+/* @var array $attributes Block attributes. */
+$attributes = wp_parse_args( $attributes );
+
+// Get the user ID from the selected user attribute.
+$selected_user = $attributes['selectedUser'] ?? 'site';
+$user_id       = Blocks::get_user_id( $selected_user );
+$button_only   = $attributes['buttonOnly'] ?? false;
+
+// Generate a unique ID for the block.
+$block_id = 'activitypub-follow-me-block-' . wp_unique_id();
+
+// Get block style information.
+$style            = wp_get_global_styles();
+$background_color = $attributes['backgroundColor'] ?? $style['color']['background'] ?? '';
+
+// Get button style from block attributes.
+$button_style = $attributes['style'] ?? array();
+
+$actor = \Activitypub\Collection\Actors::get_by_id( $user_id );
```

Comment:
we should stick with either full path (including Namespace) or define it as `use`, but not mix it.

See `use Activitypub\Blocks;`

Reply:
Fixed in f86bc55b4a794930d687c3520c05de9515b41ae8